pajbot/modules/pnsl.py

```diff
@@ -16,6 +16,13 @@ class PNSLModule(BaseModule):
     DESCRIPTION = "Run P&SL lists through the !runpnsl command"
     CATEGORY = "Moderation"
     SETTINGS = [
+        ModuleSetting(
+            key="offline_only",
+            label="Only allow the PNSL commands to be run while the stream is offline",
+            type="boolean",
+            required=True,
+            default=False,
+        ),
```

Comment:
should move the setting down in the list of settings, below `chunk_delay` at the end.